x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/AuthorizationService.java

```diff
@@ -556,11 +564,7 @@ private void handleIndexActionAuthorizationResult(
             // if this is performing multiple actions on the index, then check each of those actions.
             assert request instanceof BulkShardRequest
                 : "Action " + action + " requires " + BulkShardRequest.class + " but was " + request.getClass();
-            final AuthorizationContext authzContext = new AuthorizationContext(
-                requestInfo.getAction(),
-                authzInfo,
-                result.getIndicesAccessControl()
-            );
+            final AuthorizationContext authzContext = new AuthorizationContext(action, authzInfo, indicesAccessControl);
```

Comment:
This looks redundant, you can reuse the "parentContext" above (which is not really a "parent' in the context it has been named like that, it's a parent inside "marybeSkipChildrenActionAuthorization".

Reply:
It is. I've renamed `parentContext` to `authzContext` and removed this line as well. https://github.com/elastic/elasticsearch/pull/91886/commits/c3cc31a635753fcaf6b374b53e3dc46a0f27bdaf